src/query/functions/src/scalars/datetime.rs

```diff
@@ -275,7 +295,20 @@ fn register_number_to_timestamp(registry: &mut FunctionRegistry) {
     ) -> Value<TimestampType> {
         vectorize_with_builder_1_arg::<Int64Type, TimestampType>(|val, output, ctx| {
             match int64_to_timestamp(val) {
-                Ok(ts) => output.push(ts),
+                Ok(ts) => {
+                    let res = ts
+                        .to_timestamp(ctx.func_ctx.tz.tz)
```

Comment:
don't need to consider of tz for  `to_timestamp(int64)`, we just store the timestamp micros.

Reply:
Ok.

Reply:
Fix